tools/payments/cmd/authorize/main.go

```diff
@@ -119,6 +119,9 @@ func main() {
 				log.Fatalf("failed to read report from stdin: %v\n", err)
 			}
 
+			if len(report) < 1 {
+				log.Fatalf("report is empty: %s after reading file data: %s\n", report, f)
```

Comment:
I don't like this `Fatal` here at all (because nothing deferred will happen). But there are already quite a few others.

Technically, this is incorrect. The opened file won't be properly closed. If you want to terminate without too much hassle in this closure, better use `log.Panic` instead.

Reply:
I think this would benefit from greater attention so I've made a ticket for it. https://github.com/brave-intl/bat-go/issues/2564